R/configure.R

```diff
@@ -65,6 +69,8 @@ conan_configure <- function(method, ..., path_lib, path_bootstrap,
       assert_scalar_character(args$refs, "refs", call = rlang::current_env())
     }
     assert_scalar_character(args$policy, "policy", call = rlang::current_env())
+  } else if (method == "renv") {
+    valid_args <- NULL
   } else if (method == "auto") {
     valid_args <- NULL
```

Comment:
Not relating to this PR really - but I was expecting from the doc change above to see something like
`valid_args <- "environment"` for the auto method? On line 80, it seems "environment" is added for all methods anyway - just slightly wondering what this means.

Reply:
yeah, there's some tidying that can be done there I think once the dust settles here, I got confused on that too and churned around with changes until leaving things as they are here. The `environment` argument is only looked at in the case where `method = "auto"` but we pass it unconditionally in hipercow, *in case* we want to use the auto method. So it's always allowed but generally ignored.

Reply:
Ok - that's about what I was thinking too... One for later.